opentracing-proxy/opentracing-proxy/src/test/java/org/zalando/opentracing/proxy/intercept/span/SpanBuilderInterceptorTest.java

```diff
@@ -0,0 +1,81 @@
+package org.zalando.opentracing.proxy.intercept.span;
+
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+import io.opentracing.Tracer.SpanBuilder;
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.zalando.opentracing.proxy.base.ForwardingSpanBuilder;
+import org.zalando.opentracing.proxy.core.ProxyTracer;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers.not'](https://app.codacy.com/manual/whiskeysierra/opentracing-toolbox/pullRequest?prid=4796534)